lc-hax/Scripts/Patches/UnlimitedAmmoPatch.cs

```diff
@@ -4,11 +4,11 @@
 
 namespace Hax;
 
-[HarmonyPatch(typeof(ShotgunItem))]
-class UnlimitedAmmoPatch {
-    [HarmonyPatch(nameof(ShotgunItem.ItemActivate))]
-    static void Prefix(ref int ___shellsLoaded) => ___shellsLoaded = 2;
-
-    [HarmonyPatch(nameof(ShotgunItem.ShootGun))]
-    static void Postfix(ref int ___shellsLoaded) => ___shellsLoaded = 2;
```

Comment:
I see you removed the patch on the `ShootGun` method. Would this even work?

Reply:
Hmm.. I see.. `ItemActivate` is used to shoot the gun.

Reply:
yes, by setting ammo to 3 before `ItemActivate`, `ShootGun` will decrement it to 2, so all is good